rfhub2/cli/rfhub_importer.py

```diff
@@ -238,6 +241,24 @@ def _serialise_keywords(self, libdoc: Dict) -> Dict:
                 keyword["args"] = ""
         return keywords
 
+    def _extract_doc_from_libdoc_inits(self, inits: List) -> str:
+        return "\n" + "\n" + "\n".join([d.doc for d in inits]) if len(inits) > 0 else ""
+
+    def _contains_robot_files(self, path: Path) -> bool:
+        return (
+            len(
+                [file for file in path.glob("**/*") if file.suffix in RESOURCE_PATTERNS]
```

Comment:
it will be probably selecting multiple files that will be later discarded because it filters just based on the popular suffixes. It will get the job done as the files are later analyzed, but maybe it could be renamed to use "robot files candidates" or something similar that would indicate, that there may be wrong files identified.

Reply:
Goog idea, it's changed to that.